pkg/oauth/registry/oauthaccesstoken/strategy.go

```diff
@@ -57,6 +62,11 @@ func (s strategy) Validate(ctx kapi.Context, obj runtime.Object) field.ErrorList
 	return validationErrors
 }
 
+// ValidateUpdate validates an update
+func (s strategy) ValidateUpdate(ctx kapi.Context, obj, old runtime.Object) field.ErrorList {
+	return kvalidation.ValidateImmutableField(obj, old, &field.Path{})
```

Comment:
K.  Although this is net new and it's one less thing we have to worry
about, I don't care either way.